packages/plugin-ext/src/main/browser/view/plugin-view-widget.ts

```diff
@@ -112,6 +116,17 @@ export class PluginViewWidget extends Panel implements StatefulWidget {
         this.updateWidgetMessage();
     }
 
+    private _description: string = '';
+    get description(): string {
+        return this._description;
+    }
+
+    set description(description: string) {
+        this._description = description;
+        this.onDescriptionChange.fire();
```

Comment:
Should be moved under `this.updateWidgetDescription()`.

Reply:
Have now removed this block of code as it was redundant - I use the `setWidgetMessage` flow as an example when first writing this but it's now obvious that description is more of an upward flowing property and has no utility or path to be passed down.